File: fibonacci.R
target <- 28
repeater <- 5
answer <- 317811

fib <- function(n) {
  if (n < 2) {
    n
  } else {
    fib(n - 1) + fib(n - 2)
  }
}

for (n in 0:repeater) {
  result <- fib(target)
}

message(sprintf("R %s", result))

File: fibonacci.py
target = 28
repeat = 5
answer = 317811


def fib(n):
    if n < 2:
        return n
    else:
        return fib(n - 1) + fib(n - 2)


for n in range(0, repeat):
    result = fib(target)

print("Python " + str(result))
